host/elasticsearch_test.go

```diff
@@ -67,7 +67,15 @@ func (s *elasticsearchIntegrationSuite) SetupSuite() {
 	s.setupSuite("testdata/integration_elasticsearch_cluster.yaml")
 	s.esClient = CreateESClient(s.Suite, s.testClusterConfig.ESConfig.URL.String())
 	PutIndexTemplate(s.Suite, s.esClient, "testdata/es_index_template.json", "test-visibility-template")
-	CreateIndex(s.Suite, s.esClient, s.testClusterConfig.ESConfig.Indices[common.VisibilityAppName])
+
+	indexName := s.testClusterConfig.ESConfig.Indices[common.VisibilityAppName]
+	CreateIndex(s.Suite, s.esClient, indexName)
+
+	_, err := s.esClient.IndexPutSettings(indexName).
```

Comment:
Might be worth to make these a function, so that putIndex with maxWindowSize is guaranteed in effect after that call.